packages/fast-components-react-msft/app/app.tsx

```diff
@@ -100,12 +103,28 @@ export default class App extends React.Component<{}, IAppState> {
                     {componentFactory(examples, {...this.generateDesignSystem()})}
                 </SiteCategory>
                 <div slot={ShellSlot.infoBar}>
-                    <ColorPicker
-                        foregroundColor={this.state.foregroundColor}
-                        backgroundColor={this.state.backgroundColor}
-                        accentColor={this.state.accentColor}
-                        onColorUpdate={this.handleColorUpdate}
-                    />
+                    <div style={{display: "flex", alignItems: "center", height: "100%"}}>
+                        <Label
+                            style={{marginRight: "8px"}}
+                        >
+                            density
+                        </Label>
+                        <input
+                            type="range"
+                            name="density"
+                            defaultValue="0"
+                            min="-3"
+                            max="3"
```

Comment:
@nicholasrice might be able to give better direction on this but I believe the min should be `0.4` for now. We need to work on the density story a little to find out the proper range.